models/system/databricks-dolly-v2-12b/description.md

```diff
@@ -0,0 +1,25 @@
+The dolly-v2-12b model is a large, instruction-following language model created by Databricks, Inc. and is licensed for commercial use. It is based on EleutherAI’s Pythia-12b and was trained on a 15,000-record instruction corpus generated by Databricks employees, which was released under a CC-BY-SA license. It is not a state-of-the-art model, but it does exhibit high-quality instruction-following behavior that is not characteristic of the foundation model on which it is based. The model is available in three sizes: dolly-v2-12b, dolly-v2-7b, and dolly-v2-3b.
+
+> The above summary was generated using ChatGPT. Review the <a href="https://huggingface.co/databricks/dolly-v2-12b" target="_blank">original model card</a> to understand the data used to train the model, evaluation metrics, license, intended uses, limitations and bias before using the model.
+
+### Sample inputs and outputs (for real-time inference)
+
+```json
+{
+    "inputs": {
+        "input_string": ["My name is John and I am", "Once upon a time,"]
+    }
+}
+```
+
+#### Sample output
+```json
```

Comment:
This need to be changed

Reply:
@haritha-bendapudi Could you please test & confirm deployment + inference of the model? 

Cc: @praveenmathamsetty

Reply:
We have deployed the model present in azureml-preview-test1 on gpu and cpu skus. 
cpu deployment: https://ml.azure.com/endpoints/realtime/loganatr-ws-flkrw/detail?wsid=/subscriptions/a7772904-6f89-406e-a2d2-e3aef319fa70/resourceGroups/loganatr-rg/providers/Microsoft.MachineLearningServices/workspaces/loganatr-ws&tid=72f988bf-86f1-41af-91ab-2d7cd011db47


gpu deployment: https://ml.azure.com/endpoints/realtime/dolly-v2-12b-ep/detail?wsid=/subscriptions/a7772904-6f89-406e-a2d2-e3aef319fa70/resourceGroups/hbendapudi-eastus2-rg/providers/Microsoft.MachineLearningServices/workspaces/hbendapudi-eastus2-ws&tid=72f988bf-86f1-41af-91ab-2d7cd011db47